From 0747972bd03307c82cafde167620c0584bf56cce Mon Sep 17 00:00:00 2001
From: Jia Ma <jiam@marvell.com>
Date: Fri, 9 Jul 2021 17:50:06 -0700
Subject: [PATCH 1639/1921] soc: marvell: hw_access: add dependency
 OCTEONTX2_AF

hw_access call functions in octoentx2_af driver, and it
is compiled as module by default. So need to add this
dependency since compile will fail on other platforms without
OCTEONTX2_AF being enabled.

Signed-off-by: Jia Ma <jiam@marvell.com>
Change-Id: Ib12113dd7535b71a556c23403c1c7c0904724f9a
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/56019
Tested-by: Jia Ma <Jia.Ma@cavium.com>
Reviewed-by: Jia Ma <Jia.Ma@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/soc/marvell/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/soc/marvell/Kconfig b/drivers/soc/marvell/Kconfig
index 94378d323be0..ded83da0f813 100644
--- a/drivers/soc/marvell/Kconfig
+++ b/drivers/soc/marvell/Kconfig
@@ -141,6 +141,7 @@ config MRVL_PHY_DIAGNOSTICS
 config HW_CSR_ACCESS
 	tristate "Access HW Device Config Space"
         default m
+	depends on OCTEONTX2_AF
 	help
 	  provides an interface which enables user space utility to read from or to write
 	  to HW Device Config Space Registers, Contexts, Memory.
-- 
2.31.1

